src/test/java/org/prebid/server/bidder/openx/OpenxBidderTest.java

```diff
@@ -462,6 +465,20 @@ public void extractTargeting() {
         assertThat(openxBidder.extractTargeting(mapper.createObjectNode())).isEmpty();
     }
 
+    private static Map<String, JsonNode> stringJsonNodeMap(String key, List<String> values) {
```

Comment:
I'd like to replace both of `stringJsonNodeMap(...)` methods with something like:
```
private static Map<String, JsonNode> givenCustomParams(String key, Object value) {
    return singletonMap(key, mapper.valueToTree(value));
}
```
Since we don't need to care about the type of given value.